server/src/routes/api/v1/showsReviewsRouter.js

```diff
@@ -8,9 +8,13 @@ import cleanUserInput from "../../../services/cleanUserInput.js"
 const reviewsRouter = new express.Router({ mergeParams: true})
 
 reviewsRouter.post("/", async (req,res) => {
-  const showId = req.params.id
   const uncleanInput = req.body
+
+  const showId = req.params.id
+  const userId = req.body.userId
```

Comment:
we should be able to access the user from `req.user.id` based on the User in session, that way you can remove the `user` from your POST body